specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/preview/2019-02-01-preview/examples/CreateImageTemplate.json

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
Please add the prior version as the first commit and then check in the new version changes. That makes it possible to diff the versions in the PR

Reply:
done